Tasks/Buhayev/Task1/Task1.py

```diff
@@ -0,0 +1,85 @@
+#simplest implementation
+amount, duration, percent, factor = 20000, 5, 15, 12
+simplest_total_deposit = round(amount * ((1 + percent / 100 / factor) ** (duration * factor)),2)
+
+print(simplest_total_deposit)
+
+#simple lambda-function for calculation total amount of deposit after 5 years
+
+lambda_total_deposit = lambda amount = 20000, duration = 5, percent= 15, factor = 12: round((amount * (1 + percent / 100 / factor) ** (duration * factor)),2)
```

Comment:
it's a cool syntax just for the sake of a cool syntax, brings nothing to this particular task (but it still cool though)